app/src/main/java/eu/laramartin/medsreminder/common/CalendarUtility.java

```diff
@@ -0,0 +1,64 @@
+package eu.laramartin.medsreminder.common;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+
+import eu.laramartin.medsreminder.model.Med;
+
+public class CalendarUtility {
+
+    private static List<Integer> getDaysOfWeek(String medDays) {
+        List<Integer> weekDays = new ArrayList<>();
+        if (medDays.contains("Mo")) {
+            weekDays.add(Calendar.MONDAY);
+        }
+        if (medDays.contains("Tu")) {
+            weekDays.add(Calendar.TUESDAY);
+        }
+        if (medDays.contains("We")) {
+            weekDays.add(Calendar.WEDNESDAY);
+        }
+        if (medDays.contains("Th")) {
+            weekDays.add(Calendar.THURSDAY);
+        }
+        if (medDays.contains("Fr")) {
+            weekDays.add(Calendar.FRIDAY);
+        }
+        if (medDays.contains("Sa")) {
+            weekDays.add(Calendar.SATURDAY);
+        }
+        if (medDays.contains("Su")) {
+            weekDays.add(Calendar.SUNDAY);
+        }
+        return weekDays;
+    }
+
+    public static long timeToNextGivenDay(int targetDoW, int targetHoW, int targetMoW) {
```

Comment:
rename to `millisToNextTargetDay`
rename `targetDoW` to `targetDayOfWeek`
rename `targetHoW` to `targetHour`
rename `targetMoW` to `targetMinute`